addons/mail/static/src/core/common/message.xml

```diff
@@ -5,6 +5,7 @@
         <div t-if="message.isNotification" class="o-mail-NotificationMessage text-break mx-auto text-muted opacity-75 px-3 text-center smaller" t-on-click="onClickNotificationMessage"  t-att-class="props.className" t-ref="root">
             <i t-if="message.notificationIcon" t-attf-class="{{ message.notificationIcon }} me-1"/>
             <span class="o-mail-NotificationMessage-author d-inline" t-if="authorName and !message.body.includes(escape(authorName))" t-esc="authorName"/> <t t-out="message.body"/>
+            <span class="o-mail-Message-date o-xsmaller ms-1" t-out="message.dateSimpleWithDay"/>
```

Comment:
Some notifications content are `<p>` so this becomes 2 line notification rather than 1.
<img width="424" alt="Screenshot 2025-03-13 at 18 29 41" src="https://github.com/user-attachments/assets/4b4c2812-99d4-4a7e-88bd-638813cb5c2f" />
I'd say last `<p>` content in `o-mail-NotificationMessage` should be inline, so it looks like this:
<img width="407" alt="Screenshot 2025-03-13 at 18 30 08" src="https://github.com/user-attachments/assets/bd4ed1ed-e2c8-4377-9b97-a52f8e24c89d" />